web/src/index.js

```diff
@@ -1 +1,54 @@
-console.log('Hello world !');
+var body = document.body;
+
+var formElt = document.createElement("form");
+formElt.enctype = "application/json";
+
+var nameInputElt = document.createElement("input");
+nameInputElt.name = "name";
+
+var urlInputElt = document.createElement("input");
+urlInputElt.name = "url";
+
+var envoyerElt = document.createElement("input");
+envoyerElt.type = "submit";
+
+formElt.appendChild(nameInputElt);
+formElt.appendChild(urlInputElt);
+formElt.appendChild(envoyerElt);
+
+body.appendChild(formElt);
+
+fetch('http://localhost:8081/api/v1/item').then(function(response) {
+    if(response.ok) {
```

Comment:
I would prefer to keep failure check on top instead of wrapping the whole code inside an `if ... else` block.
It makes the code more explicit.

You can rewrite this block like this:

```
if (!response.ok) {
  // Deal with error here...
  return
}

// Deal with success here :)